datascience/tables.py

```diff
@@ -1456,12 +1459,10 @@ def sample(self, k=None, with_replacement=True, weights=None):
         job  | wage
         a    | 10
         a    | 10
-        >>> jobs.sample(k=2, weights=make_array(1, 0, 1, 0))
+        >>> jobs.sample(k=2, weights=make_array(1, 0, 1, 0)) # Weights must be length of table.
```

Comment:
I think we shouldn't put the comment at the end of this line. Usually comments in doctests where the doctest is failing is meant to be a pointer for a common pitfall - but the error here is different ("probabilities do not sum to 1") than what's in the comment

Reply:
Oh gosh yes, that was dumb!  Fixed.  Thanks.